pypgatk/tests/pypgatk_tests.py

```diff
@@ -283,6 +283,22 @@ def test_check_ensembl_database(self):
                                 'testdata/proteindb_from_ENSEMBL_VCF-clean.fa', '--add_stop_codons', '--num_aa', '6'])
         self.assertEqual(result.exit_code, 0)
 
+    # def test_validate_peptides(self):
+    #     runner = CliRunner()
+    #     result = runner.invoke(cli,
+    #                            ['validate_peptides', '--mzml_path', 'testdata',
+    #                             '--infile_name', 'testdata/380need.txt', '--outfile_name',
+    #                             'testdata/380need_AI_out.txt', '--ions_tolerance', '0.02'])
+    #     self.assertEqual(result.exit_code, 0)
+
+    def test_get_subpos(self):
```

Comment:
@DongdongdongW full tests can be perform now with the files here http://ftp.pride.ebi.ac.uk/pub/databases/pride/resources/tools/github-files/. You can add some logic in the CI/CD to download the files first before performing the tests.